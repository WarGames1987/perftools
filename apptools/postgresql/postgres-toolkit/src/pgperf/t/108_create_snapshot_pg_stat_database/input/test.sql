SELECT pgperf.create_snapshot_pg_stat_database(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database;

SELECT pgperf.create_snapshot_pg_stat_database(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database;

SELECT pgperf.create_snapshot_pg_stat_database(2);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database;

SELECT pgperf.delete_snapshot_pg_stat_database(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database;

SELECT pgperf.delete_snapshot_pg_stat_database(2);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database;

SELECT pgperf.delete_snapshot_pg_stat_database(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_database;
