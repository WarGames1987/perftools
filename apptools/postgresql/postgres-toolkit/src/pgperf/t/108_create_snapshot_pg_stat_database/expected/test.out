SELECT pgperf.create_snapshot_pg_stat_database(0);
 create_snapshot_pg_stat_database 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database;
 count 
-------
     1
(1 row)

SELECT pgperf.create_snapshot_pg_stat_database(1);
 create_snapshot_pg_stat_database 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database;
 count 
-------
     2
(1 row)

SELECT pgperf.create_snapshot_pg_stat_database(2);
 create_snapshot_pg_stat_database 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database;
 count 
-------
     3
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_database(1);
 delete_snapshot_pg_stat_database 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database;
 count 
-------
     2
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_database(2);
 delete_snapshot_pg_stat_database 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database;
 count 
-------
     1
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_database(0);
 delete_snapshot_pg_stat_database 
----------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_database;
 count 
-------
     0
(1 row)

